mbedtls-sys/build/config.rs

```diff
@@ -290,33 +322,43 @@ pub const DEFAULT_DEFINES: &'static [CDefine] = &[
     ("MBEDTLS_PLATFORM_STD_FPRINTF",                      Undefined), // default: fprintf
     ("MBEDTLS_PLATFORM_STD_PRINTF",                       Undefined), // default: printf
     ("MBEDTLS_PLATFORM_STD_SNPRINTF",                     Undefined), // default: snprintf
+    ("MBEDTLS_PLATFORM_STD_EXIT_SUCCESS",                 Undefined), // default: 0
+    ("MBEDTLS_PLATFORM_STD_EXIT_FAILURE",                 Undefined), // default: 1
+    ("MBEDTLS_PLATFORM_STD_NV_SEED_READ",                 Undefined), // default: mbedtls_platform_std_nv_seed_read
+    ("MBEDTLS_PLATFORM_STD_NV_SEED_WRITE",                Undefined), // default: mbedtls_platform_std_nv_seed_write
     ("MBEDTLS_PLATFORM_CALLOC_MACRO",                     Undefined), // default: calloc
     ("MBEDTLS_PLATFORM_FREE_MACRO",                       Undefined), // default: free
     ("MBEDTLS_PLATFORM_EXIT_MACRO",                       Undefined), // default: exit
+    ("MBEDTLS_PLATFORM_TIME_MACRO",                       Undefined), // default: time
+    ("MBEDTLS_PLATFORM_TIME_TYPE_MACRO",                  Undefined), // default: time_t
     ("MBEDTLS_PLATFORM_FPRINTF_MACRO",                    Undefined), // default: fprintf
     ("MBEDTLS_PLATFORM_PRINTF_MACRO",                     Undefined), // default: printf
     ("MBEDTLS_PLATFORM_SNPRINTF_MACRO",                   Undefined), // default: snprintf
+    ("MBEDTLS_PLATFORM_NV_SEED_READ_MACRO",               Undefined), // default: mbedtls_platform_std_nv_seed_read
+    ("MBEDTLS_PLATFORM_NV_SEED_WRITE_MACRO",              Undefined), // default: mbedtls_platform_std_nv_seed_write
     ("MBEDTLS_SSL_CACHE_DEFAULT_TIMEOUT",                 Undefined), // default: 86400
     ("MBEDTLS_SSL_CACHE_DEFAULT_MAX_ENTRIES",             Undefined), // default: 50
     ("MBEDTLS_SSL_MAX_CONTENT_LEN",                       Undefined), // default: 16384
     ("MBEDTLS_SSL_IN_CONTENT_LEN",                        Undefined), // default: 16384
     ("MBEDTLS_SSL_OUT_CONTENT_LEN",                       Undefined), // default: 16384
+    ("MBEDTLS_SSL_DTLS_MAX_BUFFERING",                    Undefined), // default: 32768
     ("MBEDTLS_SSL_DEFAULT_TICKET_LIFETIME",               Undefined), // default: 86400
     ("MBEDTLS_PSK_MAX_LEN",                               Undefined), // default: 32
     ("MBEDTLS_SSL_COOKIE_TIMEOUT",                        Undefined), // default: 60
-    ("MBEDTLS_SSL_CIPHERSUITES",                          Undefined), // no default
+    ("MBEDTLS_SSL_CIPHERSUITES",                          Undefined), // default: MBEDTLS_TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
     ("MBEDTLS_X509_MAX_INTERMEDIATE_CA",                  Undefined), // default: 8
     ("MBEDTLS_X509_MAX_FILE_PATH_LEN",                    Undefined), // default: 512
-    ("MBEDTLS_TLS_DEFAULT_ALLOW_SHA1_IN_CERTIFICATES",    Undefined),
+    ("MBEDTLS_TLS_DEFAULT_ALLOW_SHA1_IN_CERTIFICATES",    Defined),
```

Comment:
I'm not sure about this change

Reply:
If we're not supporting it now, we probably don't want to start supporting it.

Like I said, I assumed config.rs was supposed to exactly reflect config.h. If that's not the case, then we probably want to leave this turned off.